src/System.Runtime.WindowsRuntime/src/System/Threading/Tasks/AsyncInfoToTaskBridge.dotnet53.cs

```diff
@@ -198,6 +203,10 @@ private void Complete(IAsyncInfo asyncInfo, Func<IAsyncInfo, TResult> getResults
                     switch (asyncStatus)
                     {
                         case AsyncStatus.Completed:
+                            if (AsyncCausalityTracer.LoggingOn)
+                            {
+                                AsyncCausalityTracer.TraceOperationCompletion(CausalityTraceLevel.Required, base.Task.Id, AsyncCausalityStatus.Completed);
+                            }
```

Comment:
The reason is because it would be logging the completion twice.
TrySetResult goes directly to TaskFinishStageThree() so no logging happend, but TrySetException() goes to Finish()->FinishStageTwo() where the exceptional logging will be done. 
TrySetCanceled() calls Task.CancellationCleanupLogic() which logs the cancelation.